gluon-mesh-vpn-wireguard/files/lib/gluon/gluon-mesh-wireguard/checkuplink

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+interface_linklocal() {
+        local macaddr="$(ubus call network.device status '{"name": "'"$1"'"}' | jsonfilter -e '@.macaddr')"
```

Comment:
this won't work as wireguard links have not mac address so we need another solution.